src/containers/my-quizzes/create-or-edit-quiz-container/CreateOrEditQuizContainer.tsx

```diff
@@ -77,7 +76,7 @@ const CreateOrEditQuizContainer = ({
         ? t('myResourcesPage.quizzes.successEditedQuiz')
         : t('myResourcesPage.quizzes.successAddedQuiz')
     })
-    navigate(authRoutes.myResources.root.path)
+    navigate(`${authRoutes.myResources.root.path}/?tab=quizzes`)
```

Comment:
Please consider using `createUrlPath` helper from `src/utils/helper-functions.tsx`

Reply:
Thank you! Changed